Nerdbank.GitVersioning error

I had to swtich Nerdbank.GitVersioning from 3.4.165-alpha to 3.3.37 because of a build error on Ubuntu 20.04

`~/.nuget/packages/nerdbank.gitversioning/3.4.165-alpha/build/Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
~/.nuget/packages/nerdbank.gitversioning/3.4.165-alpha/build/Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: System.ArgumentException: No git repo found here. (Parameter 'workingDirectory')
~/.nuget/packages/nerdbank.gitversioning/3.4.165-alpha/build/Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Managed.ManagedGitContext..ctor(String workingDirectory, String dotGitPath, String committish)
~/.nuget/packages/nerdbank.gitversioning/3.4.165-alpha/build/Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.GitContext.Create(String path, String committish, Boolean writable)
~/.nuget/packages/nerdbank.gitversioning/3.4.165-alpha/build/Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()`

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -54,7 +54,7 @@
         <AnalysisLevel>latest</AnalysisLevel>
     </PropertyGroup>
     <ItemGroup>
-        <PackageReference Include="Nerdbank.GitVersioning" Version="3.4.165-alpha">
+        <PackageReference Include="Nerdbank.GitVersioning" Version="3.3.37">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>

